generator/src/test/resources/referenceJava/BRecord.java

```diff
@@ -0,0 +1,135 @@
+import java.util.List;
+import javax.annotation.Generated;
+import javax.annotation.Nonnull;
+import com.linkedin.data.DataMap;
+import com.linkedin.data.schema.PathSpec;
+import com.linkedin.data.schema.RecordDataSchema;
+import com.linkedin.data.schema.SchemaFormatType;
+import com.linkedin.data.template.DataTemplateUtil;
+import com.linkedin.data.template.GetMode;
+import com.linkedin.data.template.RecordTemplate;
+import com.linkedin.data.template.SetMode;
+
+
+/**
+ * 
```

Comment:
nit: fill javadoc comment out?

Reply:
this is removed (by addressing Karthik's comment)